pytest.ini

```diff
@@ -1,3 +1,5 @@
 [pytest]
-addopts = --black
-python_files = tests/*.py
\ No newline at end of file
+flake8-max-line-length = 180
```

Comment:
Is there a reason why we are diverting from the standard PEP 8 style guideline: `Maximum Line Length = 79`?

Reply:
Yes, in `tests/banks.py` line `352` you may notice the long signature uuid
`        signature="b9106148b9c6d445f6a5fe7bb54b552ac2ff639cb72e2af70f7565904120dbb2040987c6cad559d7aa3b507c8d475af9291e4faee4930b324996c7a3c0696805",`

Reply:
We can either choose to:
1. Break up the string in multiple line, such as
```python
signature="b9106148b9c6d445f6a5fe7bb54b552ac2ff639cb72e2af70f75659"
          "04120dbb2040987c6cad559d7aa3b507c8d475af9291e4faee4930b"
          "324996c7a3c0696805",
```

2. Or ignore the linting on the test files.

My preference goes out to 1.

Reply:
breaking it up like this fails the test.
If we want to do that I believe we can use 3 quotes strings and it passes the test.

`response = bank.patch_validator(
        node_id="d5356888dc9303e44ce52b1e06c3165a7759b9df1e6a6dfbd33ee1c3df1ab4d1",
        trust=76.28,
        signature="""b9106148b9c6d445f6a5fe7bb54b552
        ac2ff639cb72e2af70f7565904120dbb2040987c6cad559d7aa3b507c8d475af9291e4faee4930b324996c7a3c0696805""",
    )`

Reply:
Silly me, done now

Reply:
I will still suggest to keep it at **79**, because this is the preferred code style for Python projects. 
Also think about new junior developers joining this project in the future. We need to be a good example to them. This way they get familiar with PEP 8 and have a good reference project for their future learning.

Reply:
Sure there a lot of lines to be modified, lets keep it to 120 now get the CI/CD out, and create a separate task to restrict the line length in Codebase to **79**.

Reply:
Sure, that is good too.

Reply:
Cool, could you please approve the review ?